caravel/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -27,7 +27,7 @@ const defaultProps = {
 };
 
 
-class QueryTable extends React.Component {
+class QueryTable extends React.PureComponent {
```

Comment:
what happens when we extend `PureComponent` rather than `Component` and don't use `componentShouldUpdate` method?